portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -75,6 +78,13 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         // Get the input from the form.
         String name = getParameter(request, "name", "");
         String text = getParameter(request, "comment", "");
+
+        Entity commentEntity = new Entity("Comment");
+        commentEntity.setProperty("name", name);
+        commentEntity.setProperty("text", text);
+
+        DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
I would move this variable to the class and initialize it in the `init` method -- both the GET and POST handlers need to use it (and this service can be shared my multiple threads/uses)